app/services/curation_concerns/workflow/confirm_submission.rb

```diff
@@ -0,0 +1,30 @@
+module CurationConcerns
```

Comment:
You should not use the CurationConcerns namespace in Sufia. Use `Sufia` instead.